### PR TITLE
feat: add link to download the uniswap wallet to the homepage

### PR DESCRIPTION
**Pull request by natew**
## Description

Add link to download the uniswap wallet to the homepage

_Linear ticket:_ https://linear.app/uniswap/issue/WEB-1907/add-cta-for-uniswap-wallet-on-the-landing-page
_Slack thread:_
_Relevant docs:_

### Before

CleanShot 2023-06-02 at 11 35 07@2x

### After

CleanShot 2023-06-02 at 11 34 35@2x
CleanShot 2023-06-02 at 11 34 47@2x

### Reproducing the error

1. 

### QA (ie manual testing)

- [ ] N/A

### Automated testing

- [ ] Unit test
- [ ] Integration/E2E test

### CODE REVIEW

**Review by natew — COMMENTED** [review]
A couple notes:

- Made the hover effect use `theme.textTertiary` but maybe @infredible or others know better
- Spacing also a bit of a guess as far as the space below
- Assume we want the link not to be target="_blank" but something to consider

**Review by just-toby — APPROVED** [review]
lgtm assuming design approves

File: src/pages/Landing/index.tsx [hunk 1]
```diff
@@ -369,7 +369,18 @@ export default function Landing() {
...
               <LearnMoreArrow />
             </LearnMoreContainer>
+
+            <DownloadWalletLink href="https://wallet.uniswap.org/">
+              <svg width="20" height="20" viewBox="0 0 814 1000">
+                <path
+                  fill="currentColor"
+                  d="M788.1 340.9c-5.8 4.5-108.2 62.2-108.2 190.5 0 148.4 130.3 200.9 134.2 202.2-.6 3.2-20.7 71.9-68.7 141.9-42.8 61.6-87.5 123.1-155.5 123.1s-85.5-39.5-164-39.5c-76.5 0-103.7 40.8-165.9 40.8s-105.6-57-155.5-127C46.7 790.7 0 663 0 541.8c0-194.4 126.4-297.5 250.8-297.5 66.1 0 121.2 43.4 162.7 43.4 39.5 0 101.1-46 176.3-46 28.5 0 130.9 2.6 198.3 99.2zm-234-181.5c31.1-36.9 53.1-88.1 53.1-139.3 0-7.1-.6-14.3-1.9-20.1-50.6 1.9-110.8 33.7-147.1 75.8-28.5 32.4-55.1 83.6-55.1 135.5 0 7.8 1.3 15.6 1.9 18.1 3.2.6 8.4 1.3 13.6 1.3 45.4 0 102.5-30.4 135.5-71.3z"
+                />
+              </svg>
```
Comment (tinaszheng) [inline src/pages/Landing/index.tsx:379]: can this be moved to a separate file 🤔

File: src/pages/Landing/index.tsx [hunk 2]
```diff
@@ -393,3 +404,18 @@ export default function Landing() {
     </Trace>
   )
 }
+
+const DownloadWalletLink = styled.a`
+  display: inline-flex;
```
Comment (tinaszheng) [inline src/pages/Landing/index.tsx:409]: nice!

**Review by tinaszheng — APPROVED** [review]

**Pull request merged**